src/react/components/pages/editorPage/editorFooter.test.tsx

```diff
@@ -126,4 +126,24 @@ describe("Footer Component", () => {
         expect(onTagClickHandler).not.toBeCalled();
     });
 
+    it("componentDidUpdate check", async () => {
+        const onChangeHandler = jest.fn();
+        const onTagClickHandler = jest.fn();
+        const wrapper = createComponent({
+            tags: originalTags,
+            displayHotKeys: true,
+            onTagsChanged: onChangeHandler,
+            onTagClicked: onTagClickHandler,
+        });
+
+        wrapper.setProps({tags: [...originalTags, {color: "#808000", name: "NEWTAG"}]});
+
+        await MockFactory.flushUi();
+        wrapper.update();
+
+        const tagChild = wrapper.find("div.tag");
+
+        expect(tagChild.length).toEqual(originalTags.length + 1);
+        expect(tagChild.at(tagChild.length - 1).childAt(0).childAt(1).props().children[1]).toEqual("NEWTAG");
```

Comment:
Is this looking for the new tag in the list?  would `expect(tagChild.last().text()).toEqual("NEWTAG")` work?  This current selector is a bit difficult to follow.

Reply:
Yes, it works with ...toMatch()